nmml_about_me/about_me.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+

```

Comment:
When you make a pull request it is to indicate that you are done.

I am going to go ahead and close this and you can reopen it when you think you are completely done.

Its looking nice btw =)